news/_posts/2019-12-23-spark-3.0.0-preview2.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Preview release of Spark 3.0
+categories:
+- News
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+To enable wide-scale community testing of the upcoming Spark 3.0 release, the Apache Spark community has posted a <a href="https://archive.apache.org/dist/spark/spark-3.0.0-preview2/">Spark 3.0.0 preview2 released</a>. This preview is <b>not a stable release in terms of either API or functionality</b>, but it is meant to give the community early access to try the code that will become Spark 3.0. If you would like to test the release, please download it, and send feedback using either the <a href="https://spark.apache.org/community.html">mailing lists</a> or <a href="https://issues.apache.org/jira/browse/SPARK/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel">JIRA</a>.
+
+The Spark issue tracker already contains a list of <a href="https://issues.apache.org/jira/browse/SPARK-26078?jql=statusCategory%20%3D%20done%20AND%20project%20%3D%2012315420%20AND%20fixVersion%20%3D%2012339177%20ORDER%20BY%20priority%20DESC%2C%20key%20ASC">features in 3.0</a>.
```

Comment:
Can we change the link to a more printable list?

https://issues.apache.org/jira/sr/jira.issueviews:searchrequest-printable/temp/SearchRequest.html?jqlQuery=statusCategory+%3D+done+AND+project+%3D+12315420+AND+fixVersion+%3D+12339177+ORDER+BY+priority+DESC%2C+key+ASC&tempMax=1000